<launch>
  <!-- These are always on the Rover in all configurations -->
  
  <node pkg="nodelet" type="nodelet" name="zed_streamer"
        args="load mrover/GstWebsocketStreamerNodelet perception_nodelet_manager"
        output="screen">
    <param name="image_topic" value="/camera/left/image"/>
    <param name="port" value="8081"/>
    <param name="width" value="1280"/>
    <param name="height" value="720"/>
  </node>

  <!-- <node pkg="mrover" type="gst_websocket_streamer" name="static_streamer"
        output="screen">
    <param name="dev_path" value="/devices/platform/3610000.xhci/usb1/1-2/1-2.4/1-2.4.3"/>
    <param name="port" value="8082"/>
  </node> -->

  <node pkg="mrover" type="gst_websocket_streamer" name="mobility_streamer"
        output="screen">
    <param name="dev_path" value="/devices/platform/3610000.xhci/usb1/1-2/1-2.4/1-2.4.4"/>
    <param name="port" value="8083"/>
  </node>
</launch>
